core/server/src/main/java/alluxio/RpcUtils.java

```diff
@@ -60,8 +61,10 @@
     try {
       return callable.call();
     } catch (AlluxioException e) {
+      LOG.debug("Internal Alluxio error when running rpc", e);
       throw e.toThrift();
     } catch (IOException e) {
+      LOG.warn("I/O error when running rpc", e);
```

Comment:
I set this one to be `warn` because an I/O exception is not due to user error but possibly a transient system failure.